src/modeler/AutoRest.Swagger/JsonConverters/PathLevelParameterConverter.cs

```diff
@@ -64,8 +64,23 @@ public override bool CanConvert(System.Type objectType)
                 // Removing the common parameters to avoid serialization errors
                 jo.Remove("parameters");
             }
-            return JsonConvert.DeserializeObject<Dictionary<string, Operation>>(jo.ToString(),
-                GetSettings(serializer));
+
+            var result = new Dictionary<string, Operation>();
+
+            foreach (JProperty operation in jo.Children())
+            {
+                try
+                {
+                    result[operation.Name] = JsonConvert.DeserializeObject<Operation>(operation.Value.ToString(),
+                        GetSettings(serializer));
+                }
+                catch (JsonException)
+                {
+                    // Skip
```

Comment:
so we will silently fail for any *actual* deserialization issues in an operation, swallowing the operation without a bleep? I'd be very restrictive here, not even work with `try catch` but rather check the `operation.Name` or something for `x-` and otherwise try to deserialize without mercy.